### PR TITLE
Feature 2- Pullreqeust

### PR DESCRIPTION
**Pull request by ndstkpt**
feature-2 verify-code

### CODE REVIEW

**Comment by ndstkpt** [comment]
hi code is fine please add  proper functions as well

File: festure2.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 line1 added feature-2
+line2 added feature2
```
Comment (ndstkpt) [inline festure2.txt:2]: code is good. feature2

**Comment by ndsaravanby** [comment]
FRS2 submission merge-done

**Comment by ndsaravanby** [comment]
merge done saravanan

**Pull request closed without merging**